docs/Explanation/ADR/adr-006-license-credential.md

```diff
@@ -0,0 +1,217 @@
+| Circulation Limit | `circulationLimit`| Integer                   | 1..1  | Amount of tokens that can be issued             				   |
+
+> **Discussion Notes:**
+> Should this license also include list of all permissions for holder? e.g. mint, burn, block etc?
+
+#### License Type
+
+`licenseType` CAN TAKE multiple values based on the regulation. The examples here are based on MiCA regulation:
+
+| Code      | Name                      | Description                                  |
+| :-------- | :------------------------ | :------------------------------------------- |
+| `MICAEMI` | E-Money Token             | E-Money Token as defined under MiCA          |
+| `MICAART` | Asset Referenced Token    | Asset Referenced Token as defined under MiCA |
+
+
+### Agreement
+
+| Level 2       	| Name        	  | Type          | Mult. | Notes     									  |
+| :---------------- | :-------------- | :------------ | :---- | :---------------------------------------- |
+| Agreement Type	| `agreementType` | String        | 1..1  | `NetworkAgreement_v1.0`                   |
+| Message Families  | `msgFamilies`   | List[String]  | 1..1  | Covered message families (e.g. `TFR`, `ALL`)  |
+
+
+### Example Credential
+
+Example credential in `json-ld` format is as follows:
+
+```javascript
+{
+	"@context": [
+		"https://www.w3.org/2018/credentials/v1",
+		"https://www.cosmos.cash/2021/credentials/license/v1"
+	],
+	"id": "did:cosmos:cash:1000bb528777",
+	"type": [
+		"VerifiableCredential", 
+		"LicenseCredential"
+	],
+	"issuer": "did:sov:12345",
+	"issuanceDate": "2021-08-01T15:23:24Z",
+	"credentialSubject": {
+		"license": {
+			"licenseType": "MICAEMI", // MiCA issuer license - can fine tune this to asset backed tokens, E-Money tokens etc
+			"country": "CH",
+			"authority": "Another Financial Services Body (AFFB)",
+            "denom": "sEUR",
+            "circulationLimit": 100
+		}
+	},
+	"proof": {
+		"type": "RsaSignature2018",
+		"created": "2021-08-01T15:23:24Z",
+		"proofPurpose": "assertionMethod",
+		"verificationMethod": "https://example.edu/issuers/keys/1",
+		"jws": "eyJhbGciOiJSUzI1NiIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..." // truncated for readability
+  	}
+}
+```
+
+
+
+## Consequences
+
+### Backwards Compatibility
+
+This is the first definition of the credential. Backwards compatibility is not a concern. 
+
+### Positive
+
+This ADR offers the following benefits:
+
+- Regulatory compliance through open standards
+- It implements a Role Based Access to services on Cosmos Cash.
+- It uses Self-Sovereign Identity (SSI) standards so this means:
+  - Interoperability - for example the credential can be stored in a variety of different identity wallets
+  - Flexibility - for example, this credential can be re-used in other scenarios where token issuance is required
+- Adoption of SSI for provision of financial services offers benefits in terms of licensing and management on public platforms.
```

Comment:
we haven't talked yet about SSI, what is this? link to trusted resource or our own section?